test/import-test.ts

```diff
@@ -0,0 +1,205 @@
+import MockKintoneApiClient from "./MockKintoneApiClient";
+
+describe("import", () => {
+  const testDestDir = "testDestDir";
+  const filesToTestContent = [
+    `${testDestDir}/desktop/js/bootstrap.min.js`,
+    `${testDestDir}/desktop/js/a.js`,
+    `${testDestDir}/desktop/css/bootstrap.min.css`,
+    `${testDestDir}/desktop/css/bootstrap-reboot.min.css`,
+    `${testDestDir}/desktop/css/bootstrap-grid.min.css`,
+    `${testDestDir}/mobile/js/bootstrap.js`,
+    `${testDestDir}/mobile/js/b.js`
+  ];
+
+  const uploadFileBody = `(function() { console.log("hello"); })();`;
+
+  describe("runImport", () => {
+    let kintoneApiClient: MockKintoneApiClient;
+    let manifest: ImportCustomizeManifest;
+    let status: { retryCount: number };
+    let options: Option;
+    beforeEach(async () => {
+      kintoneApiClient = new MockKintoneApiClient(
+        "kintone",
+        "hogehoge",
+        "basicAuthUser",
+        "basicAuthPass",
+        "example.com",
+        {
+          proxy: "",
+          guestSpaceId: 0
+        }
+      );
+      manifest = {
+        app: "1"
+      };
+      status = {
+        retryCount: 0
+      };
+      options = {
+        lang: "en",
+        proxy: "",
+        guestSpaceId: 0,
+        destDir: testDestDir
+      };
+    });
+
+    afterEach(() => {
+      const handleError = (err: Error) => {
+        if (err) {
+          throw err;
+        }
+      };
+      rimraf(`${testDestDir}`, handleError);
+    });
+
+    const assertManifestContent = async (err: any, buffer: Buffer) => {
+      if (err) {
+        assert.fail(err.message);
+      }
+      const appCustomize = {
+        app: "1",
+        scope: "ALL",
+        desktop: {
+          js: [
+            "https://js.cybozu.com/vuejs/v2.5.17/vue.min.js",
+            "https://js.cybozu.com/lodash/4.17.11/lodash.min.js",
+            `${testDestDir}/desktop/js/bootstrap.min.js`,
+            `${testDestDir}/desktop/js/a.js`
+          ],
+          css: [
+            `${testDestDir}/desktop/css/bootstrap.min.css`,
+            `${testDestDir}/desktop/css/bootstrap-reboot.min.css`,
+            `${testDestDir}/desktop/css/bootstrap-grid.min.css`
+          ]
+        },
+        mobile: {
+          js: [
+            "https://js.cybozu.com/jquery/3.3.1/jquery.min.js",
+            "https://js.cybozu.com/jqueryui/1.12.1/jquery-ui.min.js",
+            `${testDestDir}/mobile/js/bootstrap.js`,
+            `${testDestDir}/mobile/js/b.js`
+          ]
+        }
+      };
+      assert.deepStrictEqual(JSON.parse(buffer.toString()), appCustomize);
+    };
+
+    const assertDownloadFileContents = async (err: any, buffer: Buffer) => {
+      if (err) {
+        assert.fail(err.message);
+      }
+      assert.strictEqual(uploadFileBody, buffer.toString());
+    };
+
+    const assertDownloadFile = (path: string) => {
+      assert.ok(fs.existsSync(path), `${path} is exists`);
+      fs.readFile(path, assertDownloadFileContents);
```

Comment:
This seems to have to use `fs.readFileSync` instead.
Because this `assertDownloadFileContents` will execute asynchronously so this assert doesn't work.